crescent/internal/registry.py

```diff
@@ -174,18 +174,16 @@ def build_commands(self) -> Sequence[AppCommand]:
                     description=unwrap(command.metadata.sub_group).description or "No Description",
                     type=OptionType.SUB_COMMAND_GROUP,
                     options=[],
-                    is_required=None,  # type: ignore
+                    is_required=None,
```

Comment:
is_required wants a bool, passing None. TBH this seems like a fishy type ignore @Lunarmagpie

Reply:
This used to be a bug with hikari. Command `option.is_required` was incorrectly typed as `bool` but it should actually be `Optional[bool]`. I think serialization was changed in the hikari release after i wrote that code so `is_required=False` now doesn't cause an issue with discord.

tldr `is_required=False` now wont make discord mad but it did when i wrote the code but its the correct way to do this now

Reply:
One of the mypy errors is that it's typed as a bool. Maybe it hasn't been released to pypi yet?

Reply:
What mypy error? Also the release is the current one crescent is using.

Reply:
The one that the workflow is throwing

<img width="1016" alt="Screen Shot 2022-06-12 at 8 38 45 AM" src="https://user-images.githubusercontent.com/55721663/173233597-3e4d425a-98d3-4dfb-8f90-d225a28bab01.png">

Reply:
Yeah, all three instances can be fixed by replacing `None` with `False`